joomla/joomla-cms
# com_installer manage view: searchtools ordering and some improvements

#### Description

This PR adds, for consistency, as it exists in so many other views in the backoffice, the order select box of the search tools to com_installer manage view.

Also solves a bug where by selecting only the Folder filter the search box hides.
###### Before PR

![com_installer-manage-beforepr](https://cloud.githubusercontent.com/assets/9630530/12992855/db920e94-d10e-11e5-9e32-6885ce57ba75.png)
###### After PR

![com_installer-manage-afterpr](https://cloud.githubusercontent.com/assets/9630530/12992852/d6b90fb2-d10e-11e5-96fe-a97389dd9f15.png)
#### How to test
1. Install latest staging and apply this patch
2. Go to Extensions -> Manage -> Manage
3. Check the ordering select box and the filter are working fine.
